internal/collector/processor/nodebatcher/node_batcher.go

```diff
@@ -172,7 +171,6 @@ type nodeBatcher struct {
 	currBatch     *batch
 	node          *commonpb.Node
 	resource      *resourcepb.Resource // TODO(skaris) remove when resource is added to span
-	spanFormat    string
```

Comment:
Why remove this?

Reply:
Why not a reference to context in the place of the field?